presto-main/src/main/java/com/facebook/presto/sql/analyzer/StatementAnalyzer.java

```diff
@@ -1606,6 +1606,7 @@ private Scope computeAndAssignOrderByScopeWithAggregation(OrderBy node, Scope ou
                     .collect(toImmutableList());
 
             Scope orderByAggregationScope = Scope.builder()
+                    .withRelationId(RelationId.anonymous())
```

Comment:
This could probably be identified, but I could not find a natural identifier. `orderBy` is already used as for `RelationId` earlier. @sopel39 any idea? I think `RelationId` should be extended to accept additional `differentiator` to distinguish scopes created from one AST node, eg `RelationId.from(orderBy, "orderByAggregationFields")`.